lib/generators/draper/templates/application_decorator.rb

```diff
@@ -0,0 +1,8 @@
+class ApplicationDecorator < Draper::Decorator
```

Comment:
That's an interesting idea. If we left in any commented out stuff, I'd want to include _all_ draper functionality that can be used (e.g. `decorates_finders`) with documentation explaining. I think I'd like to wait for this though for the time being unless you feel strongly about it.